src/atldld/dataset.py

```diff
@@ -0,0 +1,59 @@
+# The package atldld is a tool to download atlas data.
+#
+# Copyright (C) 2021 EPFL/Blue Brain Project
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+"""Everything related to section image datasets."""
+import enum
+
+
+class PlaneOfSection(enum.Enum):
+    """The plane of section in a section image dataset.
+
+    The values correspond to the IDs used by the AIBS and can be found here:
+    http://api.brain-map.org/api/v2/data/PlaneOfSection/query.json
+
+    Only a subset of values of interest are included. For the full list see the
+    URL above.
+    """
+
+    CORONAL = 1
+    SAGITTAL = 2
+
+    def __str__(self):
+        """Get the name of the plane of section."""
+        return self.name.lower()
+
+
+class ReferenceSpace(enum.Enum):
+    """The reference space of a section image.
+
+    The values correspond to the IDs used by the AIBS and can be found here:
+    http://api.brain-map.org/api/v2/data/ReferenceSpace/query.json
+
+    Only a subset of values of interest are included. For the full list see the
+    URL above.
+    """
+
+    P56 = 9
+    P56_LR_FLIPPED = 10
+
+    __names__ = {
+        P56: "P56 Brain",
+        P56_LR_FLIPPED: "P56 Brain, L/R Flipped",
+    }
```

Comment:
I guess it was you decided on the variable name `__names__`, right?  IMO dunders should be reserved to Python "internals".

Reply:
Yeah, not a fan either. But that's the only way I found to do this without adding a new enum member. I found out about the fact that dundered variables don't become enum members from the first comment under this question: https://stackoverflow.com/questions/36003273/declare-a-static-variable-in-an-enum-class

There are other related questions on StackOverflow
- here: https://stackoverflow.com/questions/12680080/python-enums-with-attributes
- and here: https://stackoverflow.com/questions/68524617/how-to-extend-the-python-class-attributes-for-enum-derived-classes

But I don't know if we want to add [`aenum`](https://github.com/ethanfurman/aenum) as a dependency.

I think something similar can be implemented with `namedtuple`:
```python
import enum
from collections import namedtuple

ReferenceSpaceType = namedtuple("ReferenceSpaceType", ("id", "name"))

class ReferenceSpace(enum.Enum):
    P56 = ReferenceSpaceType(9, "P56 Brain")
    P56_LR_FLIPPED = ReferenceSpaceType(10, "P56 Brain, L/R Flipped")


if __name__ == "__main__":
    ref_space = ReferenceSpace.P56
    print(ref_space.id)
    print(ref_space.name)
```

but the problem is it's not easy to get an enum member given an ID... I think that's exactly what [these examples](https://stackoverflow.com/a/36028867/2804645) on StackOverflow are trying to solve, but like I said, not sure what we should do...

Reply:
Interesting read! Thank you for all the references!

IMO what you are trying to do is slightly beyond what the `Enum` datatype is built for.

Personally, I am completely fine with keeping things simple and not worrying about a "nicer" `__str__`. I actually do not  find the "__names__" more informative than the actual enumeration names. Additionally, we can always refer the user to some external resource for more info.

Reply:
I went through all the different options again, and I have the impression all of them are more complicated than the current one. So I'd suggest to keep this as is for now, and if we add more functionality later we can think about improving this.